operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -122,6 +124,10 @@ local filename = 'mimir-reads.json';
       $.row('Query-frontend')
       .addPanel(
         $.panel('Requests / sec') +
+        $.panelDescription(
+          'Requests / sec',
+          $.exploreContainerLogsLink($._config.container_names.query_frontend)
```

Comment:
Can you add support to read-write mode too? Basically in this case we should query `query-frontend` and `mimir-read` containers instead of just the `query-frontend`.

_Same applies to the rest of the file._